model/riscv_sys_regs.sail

```diff
@@ -882,6 +888,24 @@ function is_fiom_active() -> bool = {
     User => (menvcfg.FIOM() | senvcfg.FIOM()) == 0b1,
   }
 }
+
+bitfield Mstateen0 : bits(64) = {
+  // controls to new state introduced by the following small extensions of Priv-1.14-era:
+  PRIV114  : 55
+}
```

Comment:
Should add spaces between these toplevel definitions

Reply:
done